test/es6/classes.js

```diff
@@ -1021,7 +1021,23 @@ var tests = [
             for (s of new ClassWithArgumentsAndCallerComputedNameGeneratorMethods().caller()) {}
             assert.areEqual('456', s, "s of new ClassWithArgumentsAndCallerComputedNameGeneratorMethods().caller() === '456'");
         }
-    }
+    },
+    {
+        name: "toString on constructor should return class declaration or expression",
+        body: function () {
+            var B = class { };
+            var A = class A extends B { constructor (...args) { super(...args); }  set x(a) { this._x = a; } set y(a) { this._y = a; } };

```

Comment:
classes.js is configured in rlexe.xml to run two scenarios: one with off:deferparse and the other with force:deferparse.